torch/quantization/_quantize_script.py

```diff
@@ -82,6 +82,8 @@ def get_scripted_qconfig_dict(qconfig_dict):
 
 def _prepare_script(model, qconfig_dict, is_dynamic):
     _check_is_script_module(model)
+    if any(map(lambda x : not isinstance(x, str), qconfig_dict.keys())):
+        raise ValueError('qconfig_dict should contain names(str) as keys.')
```

Comment:
I think this should be an illegalargument exception?

Reply:
A general check (like we have in eager mode) would be to warn if a qconfig_dict causes no change to the model, this means that the keys in the dict do not map to modules in the model, we have an error message in eager code at: https://github.com/pytorch/pytorch/blob/master/torch/quantization/quantize.py#L142